doc/conf.py

```diff
@@ -125,42 +123,50 @@ def linkcode_resolve(domain, info):
 
 # -- Options for autodoc output ------------------------------------------------
 
-autodoc_default_options = {
-    # Document all public members by default.
-    "members": None,
-    "show-inheritance": True,
-    # If set, autodoc will also generate document for the members not having docstrings:
-    "undoc-members": True,
-}
-
-autodoc_mock_imports = [
-    "fiona",
-]
-
-# Show the typehints in the description of each object instead of the signature.
-# We found this to be more readable.
-autodoc_typehints = "description"
-
-# TODO: Make this work. Why doesn't it work?
-# autodoc_type_aliases = {
-#     'AnyAsset': 'qgreenland.models.config.asset.AnyAsset',
-# }
-
-# Hide pydantic model configuration from the sphinx output.
-autodoc_pydantic_model_show_config_summary = False
-
-# Hide redundant pydantic validators summary
-autodoc_pydantic_model_show_validator_summary = False
-
-# Hide redundant field summary
-autodoc_pydantic_model_show_field_summary = False
-
-# Don't show collapsible JSONSchema model. It's too big for the PDF output. Is
-# there a way to enable it for HTML only and disable for PDF?
-autodoc_pydantic_model_show_json = False
-
-# Don't warn when a field is not serializable
-autodoc_pydantic_model_show_json_error_strategy = "coerce"
+if 'latexpdf' not in sys.argv:
```

Comment:
hacky

Reply:
We need a comment explaining the why of the hack, and maybe a TODO explaining how we think we can un-hack :)